klayout/drc/rule_decks/lvs_bjt.drc

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
+
+  #================================================
+  #--------------------LVS_BJT---------------------
+  #================================================
+
+  # Rule LVS_BJT.1: Minimum LVS_BJT enclosure of NPN or PNP Emitter COMP layers is 0um.
```

Comment:
in the doc diagram it's named: https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_10_09.html#lvs-bjt-mark-layer do we know which one is the good naming convention?

Reply:
actually the two looks different, so maybe the image is wrong?

(also is this  DRC rule checking the polygons on the `LVS_BJT` layer so that the related LVS rules can work accordingly?)

Reply:
@proppy I'm not sure what's the question here?

Reply:
`LVS_BJT` is a marker layer for LVS purposes that needs to follow certain DRC rules to be used properly in LVS. Those are the DRC rules side not the LVS side usage of that layer.

Reply:
Thanks for the clarification, then the image in the docs should probably updated? (it currently shows `DRC_BJT` as the layer name not `LVS_BJT`)

Reply:
You mean this:
https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_10_09.html

?

I believe that might have been an issue with DRM it self. We could modify the documentation. But here is correct.

Reply:
Yes, this should probably be named `LVS_BJT` not `DRC_BJT` according to what you just said?
![image](https://user-images.githubusercontent.com/5268/227151475-05688934-133c-4f59-af35-0a8c0df6e47a.png)

Reply:
@proppy Exactly, But this not related to the DRC code. Could you please approve this PR and add an issue for that on the documentation side not in this repo?

Reply:
https://github.com/google/gf180mcu-pdk/issues/118